src/entities/Proposal/templates/hiring.ts

```diff
@@ -0,0 +1,45 @@
+import { addressShortener } from '../../../helpers'
+import { HiringType, NewProposalHiring } from '../types'
+
+import { template } from './utils'
+
+type HiringConfig = NewProposalHiring & { name: string }
```

Comment:
Isn't name part of the `NewProposalHiring`? 🤔  Or is this because it's probably undefined?

Reply:
Since we are doing the `getName` check, I don't think we need this

Reply:
Yes, my bad! It's outdated